layouts/partials/head.html

```diff
@@ -13,6 +13,27 @@
     <meta name="description" content="{{ with .Site.Params.description }}{{ . }}{{ end }}">
     {{ .Hugo.Generator }}
 
+    {{ template "_internal/google_analytics_async.html" . }}
```

Comment:
Enables async script preloading by using the Hugo internal template for the async tracking snippet. It's placed in the head because it does not rely on the DOM to finish parsing before it becomes useful. The benefit here is better data as the beacon is able to call home before users bounce on mobile devices.

https://developers.google.com/analytics/devguides/collection/analyticsjs/